frontend/sozisel-app/src/apolloClient.ts

```diff
@@ -64,6 +68,18 @@ export function createApolloClient(
 
   return new ApolloClient({
     link: authLink.concat(splitLink),
-    cache: new InMemoryCache(),
+    cache: new InMemoryCache({
+      typePolicies: {
+        SessionRecording: {
+          fields: {
+            annotations: {
+              merge(_, incoming) {
```

Comment:
Add an explanation why you did this (if you forgot then this is complete annotations swap in cache instead of performing any kind of mergin)